cli/azd/main.go

```diff
@@ -39,7 +40,11 @@ func main() {
 
 	log.SetFlags(log.LstdFlags | log.Lshortfile)
 
-	if !isDebugEnabled() {
+	if isDebugEnabled() {
+		azcorelog.SetListener(func(event azcorelog.Event, msg string) {
+			log.Printf("%s: %s\n", event, msg)
```

Comment:
I would prefer not to have Azure SDK logs added by default to the azd logs.
IMHO, it should be an opt in configuration.  Something like `azd config logs azuresdkclient ON`